dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcStatementQuery.java

```diff
@@ -79,13 +80,19 @@
                                             queryFuture,
                                             rs);
                 } catch (SQLException e) {
+                    LOG.warning(() -> String.format("Query execution failed: %s", e.getMessage()));
```

Comment:
Logging a warning AND throwing an exception is not a good combination.
I would log in `FINE` level, including the actual exception (to see the stack trace).
Let the consumer of this future decide what to do with the exception.
Also never log twice in a single block with no operation in between the logs. That should always be a single log statement.

Reply:
Changed to finest level.